asset/sass/blocks/_subnavigation.scss

```diff
@@ -1,5 +1,6 @@
 .subnavigation {
-    width : $subnavigation-width;
+    width     : $subnavigation-width;
+    font-size : $font-size-small;
```

Comment:
I don't like this aligning keys/values, because:
- when we add something we _should_ realign all if the key is the longest, which makes diff less clear
- we don't have it in `asset/sass/_settings.scss`
- we resigned with aligning param types/descriptions in PHPDocs